### DISCUSSION TITLE
Segmentation fault using abaqus .inp mesh file

### DISCUSSION THREAD

**Discussion by bylore** [idaholab/moose#26501] [Q&A General] [answered]
Hi, I was using abaqus .inp mesh file in my moose app on a hpc center. And i got nothing but
**Segmentation fault (core dumped)**
And I generated the corresponding .e mesh file by --mesh-only option and then replace the .inp file with the .e file and everything went well.
Why is this? can you give my any clue?
My application version is  **snapshot-20-10-27-36490-ge1c3d90** and my system is centos 7

By the way, running --mesh-only in the hpc with **snapshot-20-10-27-36490-ge1c3d90** also got Segmentation fault (core dumped) error. And I generated the .e mesh file in my local machine with **snapshot-20-10-27-33431-g42342ed08d**

**Comment by bylore** [comment, marked as answer]
It seems that some users have reported this error. And the developers have fix this issue. [https://github.com/libMesh/libmesh/pull/3699](url)
I think this can be fixed just by update my moose to the newest version...

**Reply by GiudGiud** [reply]
That's good news thanks for searching for the solution. 

Please let us know if there is still a problem.
